docs/_source/docs/stack_config.rst

```diff
@@ -100,6 +101,48 @@ hooks
 A list of arbitrary shell or Python commands or scripts to run. Find out more
 in the :doc:`hooks` section.
 
+
+.. _launch_action:
+
+launch_action
+~~~~~~~~~~~~~
+* Resolvable: No
+* Can be inherited from StackGroup: Yes
+* Inheritance strategy: Overrides parent if set
+
+This setting determines whether or not the stack should be included or excluded when running
+``sceptre launch``. This must be one of two values: "include" or "exclude". The default value for
+this (which doesn't need to be specified) is "include". This **only** applies to the ``launch``
+command.
+
+If the ``launch_action`` is set to ``"exclude"``, it means that:
+ * If the stack does NOT exist, the Stack will not be created and will be skipped over
+ * If the stack *does* currently exist, it will be deleted.
+
+This setting is especially useful in two situations:
+
+1. You can use it to make conditional stacks that may or may not be included in a Stack Group based
+   upon some Jinja2 logic.
+2. If your CI/CD deployment process runs ``sceptre launch``, you can use this to have stacks deleted
+   when it runs. Once the stack(s) have been deleted, you can then delete the StackConfig file.
+
+For Example:
+
+.. code-block:: yaml
+
+   template:
+       path: "my/test/resources.yaml"
+
+   {% if not var.use_test_resources %}
+   launch_action: "exclude"
```

Comment:
A few responses:
* boolean params with sceptre are tricky. Because of the way we parse our vars, if you passed `--var exclude=false`, that would actually resolve to `"false"` which is actually truthy.
* `!stack_attr` doesn't have access to `var`, since `var` isn't a stack attribute; it's available for jinja rendering, so you'd have to do `{{ var.use_test_resources }}`
* Technically, it wouldn't be a global exclude. `create`, `update`, and `delete` will still act upon the stack just as normally. This is only an instruction for the broader, "smarter" `launch` command. Calling it "exclude" I think would less clear about what it actually does. I want the user to be able to explicitly act upon stacks as desired without really breaking the way those commands work. `launch`, however, is intended as a "apply my current configuration" command that already will perform creates, updates, and deletes based upon the current configuration and the deployed configuration.

The default launch_action, as I've configured it in this PR is `include`, so unless someone overrides that, it'll work the same as usual. So then the difference is between `launch_action: "exclude" # or maybe "skip"` and `exclude: true`.